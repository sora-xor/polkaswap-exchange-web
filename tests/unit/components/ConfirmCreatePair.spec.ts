// import { shallowMount, createLocalVue } from '@vue/test-utils'
// import Vuex from 'vuex'

// import ConfirmCreatePair from '@/components/ConfirmCreatePair.vue'
// import { tokens } from '../../mocks/tokens'
// import { SoramitsuElementsImport, TranslationMock } from '../../utils'

// const localVue = createLocalVue()
// localVue.use(Vuex)
// SoramitsuElementsImport(localVue)

// describe('ConfirmCreatePair.vue', () => {
//   let getters
//   let store

//   beforeEach(() => {
//     TranslationMock(ConfirmCreatePair)

//     getters = {
//       firstToken: () => tokens[0],
//       secondToken: () => tokens[1],
//       firstTokenValue: () => 12,
//       secondTokenValue: () => 12
//     }

//     store = new Vuex.Store({
//       modules: {
//         createPair: {
//           namespaced: true,
//           getters
//         }
//       }
//     })
//   })

//   it('should renders correctly', () => {
//     const wrapper = shallowMount(ConfirmCreatePair, {
//       localVue,
//       store,
//       propsData: {
//         visible: true,
//         title: 'Create a pair'
//       }
//     })
//     expect(wrapper.element).toMatchSnapshot()
//   })
// })

describe('ConfirmCreatePair test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
