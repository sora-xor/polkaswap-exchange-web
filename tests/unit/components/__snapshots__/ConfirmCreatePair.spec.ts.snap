// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmCreatePair.vue should renders correctly 1`] = `
<dialog-base-stub
  customclass=""
  title="Create a pair"
>
  <div
    class="pool-tokens-amount"
  >
    
    1000.00
  
  </div>
   
  <s-row-stub
    align="middle"
    class="pool-tokens"
    flex="true"
    gutter="0"
    justify="start"
  >
    <!---->
    
    
  
  </s-row-stub>
   
  <div
    class="output-description"
  >
    
    
  
  </div>
   
  <s-divider-stub
    contentposition="center"
    direction="horizontal"
    type="secondary"
  />
   
  <div
    class="tokens"
  >
    <s-row-stub
      align="top"
      class="token"
      flex="true"
      gutter="0"
      justify="space-between"
    >
      <s-row-stub
        align="top"
        flex="true"
        gutter="0"
        justify="start"
      >
        <!---->
         
        <span
          class="token-symbol"
        >
          XOR :
        </span>
      </s-row-stub>
       
      <div
        class="token-value"
      >
        12.00
      </div>
    </s-row-stub>
     
    <s-row-stub
      align="top"
      class="token"
      flex="true"
      gutter="0"
      justify="space-between"
    >
      <s-row-stub
        align="top"
        flex="true"
        gutter="0"
        justify="start"
      >
        <!---->
         
        <span
          class="token-symbol"
        >
          KSM :
        </span>
      </s-row-stub>
       
      <div
        class="token-value"
      >
        12.00
      </div>
    </s-row-stub>
  </div>
   
  <div
    class="pair-info"
  >
    <s-row-stub
      align="top"
      class="pair-info__line"
      flex="true"
      gutter="0"
      justify="space-between"
    >
      <div>
        
      </div>
       
      <div
        class="price"
      >
        <div>
          1 XOR = 10018.1818 KSM
        </div>
         
        <div>
          1 KSM = 0.0001 XOR
        </div>
      </div>
    </s-row-stub>
     
    <s-row-stub
      align="top"
      class="pair-info__line"
      flex="true"
      gutter="0"
      justify="space-between"
    >
      <div>
        
      </div>
       
      <div>
        1%
      </div>
    </s-row-stub>
  </div>
</dialog-base-stub>
`;
